sonic_platform_base/sonic_xcvr/api/xcvr_api.py

```diff
@@ -637,3 +637,14 @@ def get_error_description(self):
         """
         raise NotImplementedError
 
+    def set_high_power_class(self, enable):
```

Comment:
@longhuan-cisco Any reason for not defining this API in https://github.com/sonic-net/sonic-platform-common/blob/master/sonic_platform_base/sonic_xcvr/sfp_optoe_base.py too?